vm/src/import.rs

```diff
@@ -63,8 +64,13 @@ pub fn import_module(
     Ok(module)
 }
 
-fn find_source(vm: &VirtualMachine, current_path: PathBuf, name: &str) -> Result<PathBuf, String> {
-    let sys_path = vm.sys_module.get_attr("path").unwrap();
+fn find_source(
+    vm: &mut VirtualMachine,
+    current_path: PathBuf,
+    name: &str,
+) -> Result<PathBuf, String> {
+    let sys_module = &vm.sys_module;
+    let sys_path = vm.get_attribute(sys_module.clone(), "path").unwrap();
```

Comment:
This applies to all these changes, but why not just do `vm.get_attribute(vm.sys_module.clone(), ...)`?

Reply:
I thought I was running into an immutable borrow while borrowed mutably issue (the one you run into with ```vm.do_something(vm.something())```), but that doesn't happen with a simple member access.

Reply:
Fixed.